solr/contrib/ltr/src/java/org/apache/solr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -76,7 +76,14 @@
   /**
    * if the log feature query param is off features will not be logged.
    **/
-  public static final String LOG_FEATURES_QUERY_PARAM = "fvCache";
+  private static final String LOG_FEATURES_QUERY_PARAM = "fvCache";
+
+  // Check if features are requested and if the model feature store and feature-transform feature store are the same
+  public static boolean featuresRequestedFromSameStore(String modelFeatureStoreName, SolrQueryRequest req) {
```

Comment:
I find it odd that you put this function in the Transformer class since the QParser is the only one that uses it.